src/js/views/patients/actions/actions_views.js

```diff
@@ -20,11 +20,7 @@ import './actions.scss';
 
 const i18n = intl.patients.actions.actionsViews;
 
-const StatusIcons = {
-  queued: 'exclamation-circle',
-  started: 'dot-circle',
-  done: 'check-circle',
-};
+import { PatientStatusIcons } from 'js/static';
 
 const StateTemplate = hbs`<span class="action--{{ statusClass }}">{{fas statusIcon}}{{#unless isCompact}}{{ name }}{{/unless}}</span>`;
```

Comment:
it's possible but unlikely here.  Yeah.. I ran into that too today.. no error.. just nothing.. bizarre.. we probably need to catch the case in our handlebars helper.. but I'm not sure how we're going to accomplish that..  I suppose we could check against the keys in package.json... perhaps only when on a dev instance.. otherwise that seems like a potentially costly check at render.